docs/source/cloud_integration.md

```diff
@@ -49,16 +49,16 @@ To run NVTabular on the cloud using AWS, do the following:
    jupyter-lab --allow-root --ip='0.0.0.0' --NotebookApp.token='<password>'
    ```
 
-## Google Cloud Platform
+## Google Cloud Platform ##
 
-The Google Cloud Platform offers [Compute Engine instances with NVIDIA GPU support](https://cloud.google.com/compute/docs/gpus). We're using a VM with 8x NVIDIA A100 GPUs and eight local SSD-NVMe devices configured as RAID 0 to demonstrate the steps below.
+The Google Cloud Platform offers [Compute Engine instances with NVIDIA GPU support](https://cloud.google.com/compute/docs/gpus). We're using a VM with 8x NVIDIA A100 GPUs and eight local SSD-NVME devices configured as RAID 0 to demonstrate the steps below.
```

Comment:
It looks weird, but I think [`NVMe`](https://nvmexpress.org/) is actually correct here.

Reply:
@karlhigley - Ronay had called it out. You can leave it if it's correct.

Reply:
I just reverted all NVME references to NVMe.